src/themes/shared/space.ts

```diff
@@ -1,4 +1,5 @@
 export const space = {
+  xxs: '0.125rem',
```

Comment:
@lkappeler please don't forget to add it in the figma-tokens -> https://github.com/smg-automotive/figma-tokens/blob/main/tokens.json and ask the designer to update -> https://www.figma.com/file/WmNmJ7jdB0Z7tOjj516nFB/01.-AS24---Tokens?node-id=3%3A4 with the new token

Thank you 🧸